python/mlcroissant/pyproject.toml

```diff
@@ -51,7 +51,10 @@ dev = [
   "pytype",
   "torchdata",
 ]
-audio = ["librosa"]
+audio = [
+  "librosa",
+  "soxr==0.4.0b1",
```

Comment:
Can you add a comment with the fix: https://github.com/librosa/librosa/issues/1831#issuecomment-2176274560

Reply:
Sure, done.